docs-src/go/features.md

```diff
@@ -25,3 +25,4 @@ In this section you can find the following:
 - [How to Asynchronously complete an Activity](#asynchronous-activity-completion)
 - [How to register Namespaces](#namespaces)
 - [How to use custom payload conversion](#custom-payload-conversion)
+- [How to develop updates](#updates)
```

Comment:
This is a source node - so you can use the id of the node, and link magic rewrites -